doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,416 @@
+﻿Variable,Description
+cldfra2d,Instantaneous 2D (max in column) fraction
+total_albedo,Total sky albedo at top of atmosphere
+lwp_ex,Total liquid water path from explicit microphysics
+iwp_ex,Total ice water path from explicit microphysics
+lwp_fc,Total liquid water path from cloud fraction scheme
+iwp_fc,Total ice water path from cloud fraction scheme
+ALBDO_ave,Surface albedo
+DLWRF,Surface downward longwave flux
+DLWRFI,Instantanous surface downward longwave flux
+ULWRF,Surface upward longwave flux
+DSWRFItoa,Instantaneous top of atmosphere downward shortwave flux
+USWRFtoa,Instantaneous top of atmosphere upward shortwave flux
+ULWRFtoa,Instantaneous top of atmosphere upward longwave flux
+ULWRFI,Instantaneous surface upward longwave flux
+DSWRF,Averaged surface downward shortwave flux
+DSWRFI,Instantaneous surface downward shortwave flux
+USWRF,Averaged surface upward shortwave flux
+USWRFI,Instantaneous surface upward shortwave flux
+duvb_ave,UV-B downward solar flux
+cduvb_ave,Clear sky UV-B downward solar flux
+vbdsf_ave,Visible beam downward solar flux
+vddsf_ave,Visible diffuse downward solar flux
+nbdsf_ave,Near IR beam downward solar flux
+nddsf_ave,Near IR diffuse downward solar flux

```

Comment:
What does IR stand for?

Reply:
Infrared. My apologies, I thought I caught all of those